.gitignore

```diff
@@ -1,5 +1,7 @@
 /target/
+/libcnb-packaged/
 .DS_Store
 .idea
 Cargo.lock
 **/fixtures/*/target/
+**/fixtures/*/libcnb-packaged/
```

Comment:
Does the packaging happen directly in this repo, or inside the tempdir into which the fixtures are copied? If the latter, then it seems that `**/fixtures/*/libcnb-packaged/` doesn't need to be here?